### PR TITLE
feature(static template) : fix the subnavigation to the top on scroll action

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: cx-portal/src/components/pages/CompanyRoles/index.tsx [hunk 1]
```diff
@@ -64,6 +66,16 @@ export default function CompanyRoles() {
     })
   }, [url, language])
 
+  const scrollStarted = () => {
+    if (window.scrollY > 500) {
+      setTopReached(true)
+    } else if (topReached && window.scrollY <= 500) {
+      setTopReached(false)
+    }
```
Comment (oyo) [inline cx-portal/src/components/pages/CompanyRoles/index.tsx:74]: this looks strange to me - why is topReached set to true when scrollY > sth ? I would expect the opposite. Maybe rename this value to scrolledDown or something like that?
also it looks like you can write that as
`setTopReached(window.scrollY < sth.)`

**Review by oyo — CHANGES_REQUESTED** [review]
some questions

File: cx-portal/src/components/pages/CompanyRoles/index.tsx (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/CompanyRoles/index.tsx:74]: Here I'm looking to make sub navigation component to stick to the top. So have used topReached boolean to decide when to make it true. Condition is basically, if scrollY is more than 500, make it fix to top.

Comment (manojava-gk) [reply cx-portal/src/components/pages/CompanyRoles/index.tsx:74]: set statement updated as mentioned here

**Review by oyo — APPROVED** [review]
lgtm

**Pull request merged**